incydr/_users/client.py

```diff
@@ -170,36 +171,72 @@ def iter_all(
     def get_roles(
         self,
         user_id: str,
-    ) -> List[Role]:
+    ) -> List[UserRole]:
```

Comment:
should `get_roles` be changed to `get_user_roles`? (Then `get_available_roles` could be the new `get_roles` too)

Reply:
yeah, i like that change

Reply:
oh true, that would probably be a little more consistent

Reply:
I actually went with `list_user_roles()` for this rename